packages/vue/README.md

```diff
@@ -0,0 +1,125 @@
+# @synergy-design-system/vue
+
+This package provides [vue.js](https://vuejs.org/) wrappers for [Synergy Web Components](https://github.com/SickDesignSystem/synergy/tree/main/packages/components).
+
```

Comment:
This package aims for an improved UX when used in vue applications:
- Form support
- Two way data-binding for properties
- Auto-completion
- Event handling

Reply:
Done in latest version.